tasks/__init__.py

```diff
@@ -93,9 +91,10 @@ def _get_test_file_list():
 
 
 @task
-def test_selenium_with_retries(ctx, file_list, module=None):
+def test_selenium_with_retries(ctx, module=None):
     """Run group of tests on the browser defined by TEST_BUILD."""
     flake(ctx)
+    file_list = _get_test_file_list()
```

Comment:
Grab the file list at the beginning of this method since we are no longer passing it in.